pkg/i2gw/providers/gce/README.md

```diff
@@ -0,0 +1,41 @@
+# GCE Provider
+
+The project supports translating ingress-gce specific annotations.
+
+## Implementation-specific features
+
+The following implementation-specific features are supported:
+
+- There is no corresponding Gateway HTTP path type that matches ingress `ImplementationSpecific` path that contains `*`, except for `/*` which will map to `/` prefix. Please consider switch to `Prefix`, `Exact`, or `ImplementationSpecific` path without `*` before converting Ingress to Gateway.
+- For `ImplementationSpecific` paths without `*`, they directly map to `Exact` path type.
+
+| ingress `ImplementationSpecific` Path | map to Gateway Path                    |
+| ------------------------------------- | -------------------------------------- |
+| /*                                    | / Prefix                               |
+| /v1                                   | /v1 Exact                              |
+| /v1/                                  | /v1/ Exact                             |
+| /v1/*                                 | /v1 Prefix but not /v1(slash required) |
+
+## Feature list
+Currently supported:
+- [Basic Internal Ingress](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-internal-basic)
+- [Basic external Ingress](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-external-basic)
+- [Ingress with custom default backend](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-custom-default-backend)
+
+Not supported:
+ - [HTTP-to-HTTPS redirect](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-https)
+ 
+   - In GKE Ingress, Ingress using [HTTP-to-HTTPS redirect](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#https_redirect) is provisioned with FrontendConfig, so it is only [supported by External Ingress][https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#associating_frontendconfig_with_your_ingress] only.
+   However, this feature is [not supported](https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-gateways#configure_http-to-https_redirects) in the corresponding GKE GatewayClass `gke-l7-gxlb`.
+
+TBD(Mostly BackendConfig compatiblity):
+ - [Ingress with custom HTTP health check](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-custom-http-health-check)
+ - [IAP enabled ingress](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-iap)
+ - [Google Cloud Armor enabled ingress](https://github.com/GoogleCloudPlatform/gke-networking-recipes/blob/main/ingress/single-cluster/ingress-cloudarmor/README.md)
```

Comment:
I think both of these can be handled with [GCPBackendPolicy](https://cloud.google.com/kubernetes-engine/docs/how-to/configure-gateway-resources#configure_cloud_armor)

Reply:
Thanks for the pointer! Will keep this in mind when creating PRs for this.